src/ctelnet.cpp

```diff
@@ -1356,13 +1359,19 @@ void cTelnet::setGMCPVariables(const QByteArray& msg)
         connect(reply, &QNetworkReply::downloadProgress, this, &cTelnet::setDownloadProgress);
         mpProgressDialog->show();
         return;
-    } else if (msg.startsWith(QStringLiteral("Client.Map"))) {
+    } else if (packageMessage.startsWith(QStringLiteral("Client.Map"))) {
```

Comment:
\<aside>`QString::startsWith(const QString, ...)` is one of those methods that has a `QLatin1String` overload so for a fixed ASCII string I guess it would be okay to use it instead of a `QStringLiteral`?\</aside>